nav2_costmap_2d/include/nav2_costmap_2d/nav2_3d_static_layer.hpp

```diff
@@ -0,0 +1,70 @@
+//
+// Created by sun on 2020/10/10.
+//
+
+#ifndef NAV2_3D_STATIC_LAYER_NAV2_3D_STATIC_LAYER_HPP
+#define NAV2_3D_STATIC_LAYER_NAV2_3D_STATIC_LAYER_HPP
+
+#include "nav2_costmap_2d/costmap_2d.hpp"
+#include "nav2_costmap_2d/costmap_layer.hpp"
+#include "nav2_costmap_2d/layer.hpp"
+#include "nav2_costmap_2d/layered_costmap.hpp"
+#include "nav2_costmap_2d/obstacle_layer.hpp"
+
+namespace nav2_costmap_2d  // TODO rename ns to nav2_costmap_2d
```

Comment:
Remove or resolve TODOs, we don't allow for TODOs to be introduced into the codebase